### PR TITLE
testing/perl-net-mpd: new aport

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: testing/perl-net-mpd/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+depends="$cpandepends mpd"
+makedepends="perl-dev $cpanmakedepends"
+checkdepends="$cpancheckdepends"
+subpackages="$pkgname-doc"
+source="https://cpan.metacpan.org/authors/id/A/AB/ABERNDT/Net-MPD-$pkgver.tar.gz"
+builddir="$srcdir/$_pkgreal-$pkgver"
+
+build() {
+	export PERL_MM_USE_DEFAULT=1 PERL_AUTOINSTALL="--skipdeps" \
+    PERL_MM_OPT="INSTALLDIRS=vendor DESTDIR='$pkgdir'" \
```
Comment (Ikke) [inline testing/perl-net-mpd/APKBUILD]: Inconsistent indentation, please use tabs here.

File: testing/perl-net-mpd/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# Maintainer: Leo <thinkabit.ukim@gmail.com>
+pkgname=perl-net-mpd
+_pkgreal=Net-MPD
+pkgver=0.07
+pkgrel=0
+pkgdesc="Communicate with an MPD server"
+url="https://metacpan.org/release/Net-MPD/"
+arch="noarch"
+license="MIT"
+cpandepends=""
```
Comment (Ikke) [inline testing/perl-net-mpd/APKBUILD]: Empty variables can be removed to reduce clutter.

File: testing/perl-net-mpd/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+cpanmakedepends="perl-module-build-tiny"
+cpancheckdepends=""
+depends="$cpandepends mpd"
+makedepends="perl-dev $cpanmakedepends"
+checkdepends="$cpancheckdepends"
```
Comment (Ikke) [inline testing/perl-net-mpd/APKBUILD]: Because `cpancheckdepends` is empty, this can be removed as well.

Comment (maxice8) [reply testing/perl-net-mpd/APKBUILD]: done

**Pull request closed without merging**